src/data/factored_vocab.cpp

```diff
@@ -655,6 +655,34 @@ std::string FactoredVocab::surfaceForm(const Words& sentence) const /*override f
   return res;
 }
 
+size_t FactoredVocab::getTotNumberFactors() const {
```

Comment:
I think getTotalNumberFactors would have been clear and not much more space.  Or even getTotalFactorCount

Reply:
changed to getTotalFactorCount()